frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -67,106 +201,122 @@ export function TrainAiModelTab({ onClose }: { onClose: () => void }) {
+            .filter((layer) => layer.typ === "Volume")
+            .map((layer) => layer.name),
+        ]);
+        const segmentationLayers: Array<{ name: string }> = segmentationLayerNames.map(
+          (layerName) => ({
+            name: layerName,
+          }),
+        );
+
+        const fixedSelectedLayer = segmentationLayers.length === 1 ? segmentationLayers[0] : null;
+
+        // Remove uint24 color layers because they cannot be trained on currently
+        const colorLayers = getColorLayers(dataset).filter(
+          (layer) => layer.elementClass !== "uint24",
+        );
+        const annotationId = "id" in annotation ? annotation.id : annotation.annotationId;
+        return (
+          <Row key={annotationId} gutter={8}>
+            <Col span={8}>
+              <FormItem
+                hasFeedback
+                name={["trainingAnnotations", idx, "annotationId"]}
+                label="Annotation ID"
+                initialValue={annotationId}
+              >
+                <Input disabled />
+              </FormItem>
+            </Col>
+            <Col span={8}>
+              <FormItem
+                hasFeedback
+                name={["trainingAnnotations", idx, "imageDataLayer"]}
+                label="Image Data Layer"
+                initialValue={colorLayers.length > 0 ? colorLayers[0].name : undefined}
+                rules={[
+                  {
+                    required: true,
+                    message: "Please select a layer whose image data should be used for training.",
+                  },
+                ]}
+              >
+                <LayerSelection
+                  layers={colorLayers}
+                  getReadableNameForLayer={(layer) => layer.name}
+                  fixedLayerName={colorLayers.length === 1 ? colorLayers[0].name : undefined}
+                  style={{ width: "100%" }}
+                />
+              </FormItem>
+            </Col>
+            <Col span={8}>
+              <LayerSelectionFormItem
+                name={["trainingAnnotations", idx, "layerName"]}
+                chooseSegmentationLayer
+                layers={segmentationLayers}
+                getReadableNameForLayer={(layer) => {
+                  return layer.name;
+                }}
+                fixedLayerName={fixedSelectedLayer?.name || undefined}
+                label="Groundtruth Layer"
+              />
+            </Col>
+          </Row>
+        );
+      })}
+
+      <AiModelCommentFormItem />
       <CollapsibleWorkflowYamlEditor
         isActive={useCustomWorkflow}
         setActive={setUseCustomWorkflow}
       />
 
-      <FormItem hasFeedback name="dummy" label="Training Data">
-        <div>
-          {userBoundingBoxes.length} bounding boxes ({formatVoxels(bboxesVoxelCount)})
-        </div>
-      </FormItem>
+      {userBoundingBoxes != null ? (
+        <FormItem hasFeedback name="dummy" label="Training Data">
```

Comment:
You did not introduce it, but do you know whether there's a good reason for that `name="dummy"` or should it be replaced by something more descriptive?

Reply:
The name field apparently be removed without any errors. :shrug: